### ISSUE TITLE
Is it possible to update the step by step process for atomic name trading

### ISSUE THREAD

**Issue by browebhx** [namecoin/namecoin-core#416]
The process described below is no longer working since there has been an update with createrawtransaction.
Can anyone describe the correct  process to complete these atomic trades.

https://www.namecoin.org/docs/name-owners/atomic-name-trading/

Thank you.

**Comment by yanmaani** [comment]
A rewrite is planned, but have there been any changes to createrawtransaction?

Those commands are for the linux shell, you can't run them in the Namecoin CLI.

What is the change to createrawtransaction?

**Comment by yanmaani** [comment]
Also, this is the wrong repo; it should go in https://github.com/namecoin/namecoin.org/issues.

**Comment by browebhx** [comment]
Thank you, I think createrawtransaction does not allow name operations anymore (since 2017?) and is back to being similar to the Bitcoin command. namerawtransaction was introduced so two steps are needed to complete an atomic name trade ( from what I understand) Unless this has changed with namepsbt ... Hard to follow and figure out how to achieve an atomic name trade now for people like me catching on now. Should I repost in namecoin.org/issues ?

**Comment by yanmaani** [comment]
Oh yeah, you're right (https://www.namecoin.org/2017/09/13/raw-tx-api-refactoring.html)

You can use PSBT too, but it's not necessary.

Yes, this should go under namecoin.org, but it's a dupe of namecoin/namecoin.org#315 so the discussion should just go in there instead. As I recall it, the process is something like this:

1) Seller creates a transaction with 1 input (the name) and 2 outputs: one 0.01 NMC (to buyer's address) and one with whatever amount of money he's going to be paying (to his address)
2) Seller uses namerawtransaction to make the 0.01 NMC output into a name output for the desired name
3) Buyer runs fundrawtransaction to automatically add enough money from your wallet to buy the name
4) Buyer signs
5) Seller signs
6) Broadcast, done

I'm not 100% though and it might be a good idea to try it out on testnet first or with a low-value name. 

Anyway, this discussion should continue in namecoin/namecoin.org#315, since it's not an issue with the namecoin software itself

**Comment by browebhx** [comment]
I was missing the fundrawtransaction. I will try it out. 
Thank you.
